### PR TITLE
feat: blog post on Ory Manifesto

### PR DESCRIPTION
**Pull request by marneysof**
may have some errors in formatting.

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [x] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [x] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

File: src/markdown/blog/orymanifesto.mdx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+overline: >
+   A distillation of the key beliefs that underpin Ory as an organization.
+
+category: Blog post
+
+teaser: >
+   A distillation of the key beliefs that underpin Ory as an organization.
+---
+
+##The Ory Manifesto
```
Comment (vinckr) [inline src/markdown/blog/orymanifesto.mdx:24]:
```suggestion
## The Ory Manifesto
```
Headlines are all broken, you need to leave a space and fix this for all headlines.

File: src/markdown/blog/orymanifesto.mdx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+seo:
+  description: >
+    A distillation of the key beliefs that underpin Ory as an organization.
+  title: >
+    The Ory Manifesto
+
+publishedAt: '2021-12-08'
+author: marneysof
+overline: >
+   A distillation of the key beliefs that underpin Ory as an organization.
```
Comment (vinckr) [inline src/markdown/blog/orymanifesto.mdx:16]: Overline is too long

**Review by vinckr — CHANGES_REQUESTED** [review]

**Comment by jfcurran** [comment]
Thank you @vinckr - it's good to keep an eye on these things

File: src/markdown/blog/orymanifesto.mdx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+overline: >
+   The Ory ethos, in a nutshell.
+
+category: Blog post
+
+teaser: >
+   A distillation of the key beliefs that underpin Ory as an organization.
+---
+
+## The Ory Manifesto
```
Comment (BlueAcidFrog) [inline src/markdown/blog/orymanifesto.mdx:24]:
```suggestion
# The Ory Manifesto
```

File: src/markdown/blog/orymanifesto.mdx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  The Ory Manifesto
+
+seo:
+  description: >
+    A distillation of the key beliefs that underpin Ory as an organization.
+  title: >
+    The Ory Manifesto
+
+publishedAt: '2021-12-08'
+author: marneysof
```
Comment (BlueAcidFrog) [inline src/markdown/blog/orymanifesto.mdx:14]:
```suggestion
author: Ory Team 
```

**Review by BlueAcidFrog — APPROVED** [review]

File: src/markdown/blog/orymanifesto.mdx (hunk 4 above)
Comment (BlueAcidFrog) [reply src/markdown/blog/orymanifesto.mdx:14]: We should communicate our Ethos as if it comes from Ory, each and every one of us, like a unite.

File: src/markdown/blog/orymanifesto.mdx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  The Ory Manifesto
+
+seo:
+  description: >
+    A distillation of the key beliefs that underpin Ory as an organization.
+  title: >
+    The Ory Manifesto
+
+publishedAt: '2021-12-08'
+author: The Ory Team
```
Comment (marneysof) [inline src/markdown/blog/orymanifesto.mdx:14]:
```suggestion
author: oryteam
```

Comment (marneysof) [reply src/markdown/blog/orymanifesto.mdx:14]: this is the accurate author.

File: src/markdown/blog/orymanifesto.mdx (hunk 3 above)
Comment (marneysof) [reply src/markdown/blog/orymanifesto.mdx:24]: We already have an H1 as the title so everything in the blog post should be H2.

**Comment by aeneasr** [comment]
Good to go?

**Comment by aeneasr** [comment]
@vinckr @BlueAcidFrog

**Comment by BlueAcidFrog** [comment]
Good to go!

**Review by aeneasr — APPROVED** [review]

**Pull request merged**